R/batch.R

```diff
@@ -25,7 +25,9 @@ batch <- function(
     batchman.verbose = isTRUE(interactive()),
     stop = FALSE,
     retry = 0,
-    sleep = 0
+    sleep = 0,
+    ncores = parallel::detectCores(),
+    parallel = TRUE
```

Comment:
Don't make it parallel by default.  I'd rather it be an option people can intentionally enable.  People shouldn't be doing parallel things unless they know what they're doing.